site-src/implementations.md

```diff
@@ -5,8 +5,112 @@ and provides status and resource references for them.
+
+* **Conformant** implementations - these implementations have submitted at least
+  one conformance report that has passes for:
+
+  * All core conformance tests for at least one combination of Route type and
+    Profile
+  * All claimed Extended features
+  
+  for one of the two (2) most recent Gateway API releases.
+  So, it's conformant to support Mesh + HTTPRoute, or Gateway + HTTPRoute, or
+  Gateway + TLSRoute, or Gateway + Mesh + HTTPRoute, plus any extended features
+  the implementation claims. But implementaions _must_ support at least one
+  Profile and one Route type in that profile, and must pass all Core conformance
+  tests for that Profile and Route type in addition to all claimed Extended
+  features.
+* **Partially Conformant** implementations - these implementations are aiming for
+  full conformance but are not currently achieving it. They have submitted
+  at least one conformance report passing some of the tests to be Conformant (as
+  above) for one of the three (3) most recent Gateway API releases. Note that
+  the requirements to be considered "partially conformant" may be tightened in
+  a future release of Gateway API.
+* **Stale** implementations - these implementations may not be being actively
+  developed and will be removed from this page on the next page review unless
+  they submit a conformance report moving them to one of the other categories.
+  Page reviews are performed at least one month after every Gateway API release,
+  with the first being performed after the release of Gateway API v1.3, in late
+  June 2025. This category will itself be removed in the Gateway API v1.5
+  review process, due in mid-2026.
+
+Implementations also generally fall into two categories:
+
+* **Gateway** controllers reconcile the Gateway resource and are intended to
+handle north-south traffic, mainly concerned with coming from outside the
+cluster to inside.
+* **Mesh** controllers reconcile Service resources with HTTPRoutes attached
+and are intended to handle east-west traffic, within the same cluster or
+set of clusters.
+
+Also listed on this page are **integrations**, which are other software
+projects that are able to make use of Gateway API resources to perform
+other functions (like managing DNS or creating certificates).
+
+!!! note
+    This page contains links to third party projects that provide functionality
+    required for Gateway API to work. The Gateway API project authors aren't
+    responsible for these projects, which are listed alphabetically within their
+    class.
+
+## Adding new entries
+
+Implementations are free to make a PR to add their entry to this page; however,
+in order to meet the requirements for being Partially Conformant or Conformant,
+the implementation must have had a conformance report submission PR merged.
+
+Part of the review process for new additions to this page is that a maintainer
+will check the conformance level and verify the state.
+
+## Page Review Policy
```

Comment:
I'd recommend pushing these sections to the bottom of the page since ~99% of readers of this page will just be trying to find an implementation to use, and these particular sections are primarily targeting implementations.

Reply:
I went back and forth on this, seems good, done.